autogpt/commands/file_operations.py

```diff
@@ -204,32 +213,63 @@ def ingest_file(
             "description": "The text to write to the file",
             "required": True,
         },
+        "if_exists": {
+            "type": "string",
```

Comment:
Best to implement Enum parameter type support for this

Reply:
Reply